.circleci/config.yml

```diff
@@ -11,12 +11,13 @@ jobs:
     steps:
       - checkout
       - python/load-cache
-      - run: pip uninstall -y pytest codecov  # temporary override for broken cache
-      - run: pip install codecov pytest tqdm scikit-learn
+      - run: pip uninstall -y pytest codecov
+      # note: uninstall is required because otherwise circleci cache will lose track of pytest/codecov executables
+      - run: pip install pytest pytest-forked pytest-asyncio codecov tqdm scikit-learn
       - python/install-deps
       - python/save-cache
       - run:
-          command: pip install -e .
+          command: pip install --force-reinstall -e .
```

Comment:
this is now required due to circleci's updated caching: otherwise it may skip installation if it already has hivemind installed